Sources/XcbeautifyLib/Renderers/GitHubActionsRenderer.swift

```diff
@@ -5,6 +5,7 @@ struct GitHubActionsRenderer: OutputRendering {
         case notice
         case warning
         case error
+        case info
```

Comment:
Sorry– I think I said to use `info` in my previous comment. `info` should simply use `notice`. AFAIK, `info` is not a valid annotation.